slimta/logging/http.py

```diff
@@ -24,10 +24,11 @@
 
 """
 
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
```

Comment:
I'm not crazy about all the `str()` calls in this file. Would it work better to just remove `unicode_literals` and explicitly quote unicode strings with `u' '` instead?